.github/workflows/layer_govcloud.yml

```
@@ -0,0 +1,141 @@
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: choice
+        options:
+          - Gamma
+          - Prod
+        default: Gamma
+        required: true
+      version:
+        description: Layer version to duplicate
+        type: number
+        required: true
+
+name: Layer Deployment (GovCloud)
+run-name: Layer Deployment (GovCloud) - ${{ inputs.environment }}
+
+jobs:
+  stash:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      matrix:
+        layer:
+          - AWSLambdaPowertoolsPythonV3-python38
+          - AWSLambdaPowertoolsPythonV3-python39
+          - AWSLambdaPowertoolsPythonV3-python310
+          - AWSLambdaPowertoolsPythonV3-python311
+          - AWSLambdaPowertoolsPythonV3-python312
+        arch:
+          - arm64
+          - x86
```

```suggestion
          - x86_64
```

I understand you are still testing, but please consider changing it to the new name. We will fix the layer name before launching GovCloud.

---

I'll ignore this until that issue is resolved.